### PR TITLE
Script to update Fuel UX dependent projects

### PR DESCRIPTION
**Pull request by cormacmccarthy**
Adds script to update dependencies in all projects that rely upon fuel ux and lightning theme.

### CODE REVIEW

File: update-dependencies.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+					console.log('');
+					reject(project, release, lightningThemeRelease);
+				}
+			}
+		);
+	});
+}
+
+const updateFuelUX = (project) => {
+	const fueluxRelease = '3.15.8';
```
Comment (swilliamset) [inline update-dependencies.js]: Hardcoded version numbers. Should pull most recent tags from project.

File: update-dependencies.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+			async: false,
+			shell: '/bin/bash'
+		};
+
+		const cd = `cd ../${project}`
+
+		const packageFUX = String.raw`sed -i '' 's/\(fuelux": "\)[\^\>\=\~\<0-9.x]*\("\)/fuelux": "${release}"/g' ./package.json`;
+		const bowerFUX = String.raw`sed -i '' 's/\(fuelux": "\)[\^\>\=\~\<0-9.x]*\("\)/fuelux": "${release}"/g' ./bower.json`;
+
+		const packageFLTPrefix = String.raw`"fuelux-lightning-theme": "git+ssh:\/\/git@github.exacttarget.com:uxarchitecture\/fuelux-lightning-theme.git\#`;
```
Comment (swilliamset) [inline update-dependencies.js]: Lots of internal references. @futuremint is this a problem? should this tooling go somewhere else?

File: update-dependencies.js [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+const prepBranch = (project, release, lightningThemeRelease) => {
+	return new Promise((resolve, reject) => {
+		const execOpts = {
+			silent: true,
+			async: false,
+			shell: '/bin/bash'
+		};
+
+		const cd = `cd ../${project}`
+		const gitstuff = `git stash && git checkout master && git pull upstream master && git status`
```
Comment (swilliamset) [inline update-dependencies.js]: feature assumptions:
- parent project on "upstream" origin
- primary branch is "master"

could be modified to use a prompt but not necessary for first pass

File: update-dependencies.js (hunk 1 above)
Comment (cmcculloh) [inline update-dependencies.js]: bah. Meant to make those dynamic.

File: RELEASE.md [hunk 4]
```diff
@@ -65,7 +65,7 @@ You will need to authorize your machine to do npm publishing using `npm adduser`
       - [ ] 4b. Run `bower update`. 
       - [ ] 4c. Commit
       - [ ] 4d. Push to `upstream`.
-- [ ] 6. Run `node update-dependencies.js`
+- [ ] 6. Run `node update-dependencies.js [fuelux release #] [lightning theme release #]`
```
Comment (swilliamset) [inline RELEASE.md]: This isn't user friendly. Either we need to include a flag pattern `node app --flag value --flag2 value2` or break this into prompts in the script.

**Comment by cormacmccarthy** [comment]
This doesn't belong in this repo.

**Pull request closed without merging**